test/units/testsuite-58.sh

```diff
@@ -160,12 +160,24 @@ last-lba: 2097118
 $imgs/zzz1 : start=        2048, size=     1775576, type=933AC7E1-2EB4-4F13-B844-0E14E2AEF915, uuid=4980595D-D74A-483A-AA9E-9903879A0EE5, name=\"home-first\", attrs=\"GUID:59\"
 $imgs/zzz2 : start=     1777624, size=      131072, type=0657FD6D-A4AB-43C4-84E5-0933C84B4F4F, uuid=78C92DB8-3D2B-4823-B0DC-792B78F66F1E, name=\"swap\""
 
+    systemd-repart --offline="$OFFLINE" \
+                    --definitions="$defs" \
+                   --empty=create \
+                   --size=50M \
+                   --seed="$seed" \
+                   --include-partitions=root,home \
+                   "$imgs/qqq"
+
+    sfdisk -d "$imgs/qqq" | grep -v -e 'sector-size' -e '^$'
+
     systemd-repart --offline="$OFFLINE" \
                    --empty=create \
                    --size=1G \
                    --dry-run=no \
                    --seed="$seed" \
-                   --copy-from="$imgs/zzz" \
+                   --definitions "" \
+                   --copy-from="$imgs/qqq" \
```

Comment:
zzz?

Reply:
but then it's the same one used twice?

Reply:
Yeah, we just copy.the partitions from that one twice, that exercises the logic two times.